README.md

```diff
@@ -1,2 +1,61 @@
-# html-css-capstone
-A project for completion of HTML and CSS block at Microverse
+<h2>Psych Conference</h2>
+
+![Screenshot (45)](https://user-images.githubusercontent.com/69966775/124003051-63747900-d9ce-11eb-830e-f6d0c6c5144c.png)
+
+<h6>First Capstone Project</h6>
```

Comment:
- [ ] Although your Readme looks nice, it will be great if you can add a couple of sentences to describe the project. You may tell us what the project is all about and why you decided to use this context.

Reply:
Done! 💯